pkg/kubectl/cmd/get.go

```diff
@@ -375,6 +375,14 @@ func RunGet(f *cmdutil.Factory, out io.Writer, errOut io.Writer, cmd *cobra.Comm
 	if err != nil {
 		return err
 	}
+	if sorting == "" {
+		if len(objs) > 0 {
+			_, ok := objs[0].(*api.Event)
```

Comment:
@fabianofranz it fails miserably. I was thinking this was a quick fix that would solve the underlying issue, but i was wrong.